Dockerfile.blackhole-ui

```diff
@@ -0,0 +1,46 @@
+# Frontend build stage
+FROM node:18-slim AS frontend-builder
+WORKDIR /app
+
+# Copy package files first for better caching
+COPY client/package*.json ./
+RUN npm install
+
+# Copy the rest of the client code
+COPY client/ ./
+
+# Build the frontend first
+RUN npm run build
+
+# Create directory structure for mediainfo images
+RUN mkdir -p /app/build/static/images/mediainfo/codec \
+    /app/build/static/images/mediainfo/edition \
+    /app/build/static/images/mediainfo/resolution
+
+# Copy mediainfo images to the correct location and verify
+COPY client/src/images/mediainfo/codec/* /app/build/static/images/mediainfo/codec/
+COPY client/src/images/mediainfo/edition/* /app/build/static/images/mediainfo/edition/
+COPY client/src/images/mediainfo/resolution/* /app/build/static/images/mediainfo/resolution/
+
+# Python application stage
+FROM python:3.9-slim
+LABEL org.opencontainers.image.source="https://github.com/westsurname/scripts"
+LABEL org.opencontainers.image.description="Docker image for the blackhole service"
+
+ARG SERVICE_NAME=blackhole
```

Comment:
Shouldn't this have it's own service with it's own python dependencies?

Reply:
Wait this is a duplicate of blackhole just with the ui? Why have a separate docker image for it in that case?

Reply:
Ah the gitignore may have skipped the updated dependencies. I’ll check this

Reply:
Fixed, edited the Dockerfile.blackhole